### PR TITLE
Added an option to toggle debug mode

### PR DESCRIPTION
**Pull request by IvanKalinin**
Added an option to toggle debug mode in babel-preset-env by using debug variable

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,10 +24,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var transpileTargets = (options && options.targets) ||
     buildTargets(options || {});
 
+  var debug = options.debug !== undefined ? options.debug : true;
```
Comment (ljharb) [inline index.js]: i think it'd actually be better to default it to `false` - ie, `var debug = typeof options.debug === 'boolean' ? ‼options.debug : false;`

Comment (IvanKalinin) [inline index.js]: I've switched to false by default.

But I don't really get why should we use `!!` before `options.debug`. After all if type of options.debug is **boolean** then the value is (bool) `true` or (bool) `false`

File: index.js [hunk 2]
```diff
@@ -24,10 +24,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var transpileTargets = (options && options.targets) ||
     buildTargets(options || {});
 
+  var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
```
Comment (1pete) [inline index.js]: If type of `options.debug` is boolean, it is redundant to double negate it, isn't it?

Comment (ljharb) [inline index.js]: Sure, that's a fair point. Doesn't hurt tho.

**Pull request merged**